ng2/6-readjsonfile/web/index.html

```diff
@@ -18,6 +18,6 @@
     <script defer src="packages/browser/dart.js"></script>
```

Comment:
There are a few differences with the template that I applied to my copy:

* UTF-8 -> utf-8
* stylesheet link below the scripts
* maybe more

Perhaps we can avoid these changes for now, since they have to be duplicated in so many places (and affect the text) but file a bug about them, in case this page sticks around.